modules/Bio/EnsEMBL/Compara/PipeConfig/Parts/OrthologQMAlignment.pm

```diff
@@ -0,0 +1,172 @@
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::OrthologQM::PrepareOrthologs',
+            -parameters => {
+                'hashed_mlss_id'            => '#expr(dir_revhash(#orth_mlss_id#))expr#',
+                'homology_flatfile'         => '#homology_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.homologies.tsv',
+                'homology_mapping_flatfile' => '#homology_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.homology_id_map.tsv',
+            },
+            -analysis_capacity  =>  50,  # use per-analysis limiter
+            -flow_into => {
+                # these analyses will write to the same file, so a semaphore is required to prevent clashes
+                '3->A' => [ 'reuse_wga_score' ],
+                'A->2' => { 'calculate_wga_coverage' => INPUT_PLUS() },
+            },
+            -rc_name  => '2Gb_job',
+        },
+
+        {   -logic_name => 'calculate_wga_coverage',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::OrthologQM::CalculateWGACoverage',
+            -hive_capacity => 30,
+            -batch_size => 10,
+            -flow_into  => {
+                3 => [ '?table_name=ortholog_quality' ],
+                2 => [ 'assign_wga_coverage_score' ],
+            },
+            -rc_name => '2Gb_job',
+        },
+
+        {   -logic_name => 'assign_wga_coverage_score',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::OrthologQM::AssignQualityScore',
+            -parameters => {
+                'hashed_mlss_id' => '#expr(dir_revhash(#orth_mlss_id#))expr#',
+                'output_file'    => '#wga_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.wga.tsv',
+                'reuse_file'     => '#wga_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.wga_reuse.tsv',
+            },
+            -hive_capacity => 400,
+        },
+
+        {   -logic_name => 'reuse_wga_score',
+            -module     => 'Bio::EnsEMBL::Compara::RunnableDB::OrthologQM::ReuseWGAScore',
+            -parameters => {
+                'hashed_mlss_id'            => '#expr(dir_revhash(#orth_mlss_id#))expr#',
+                'previous_wga_file'         => '#prev_wga_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.wga.tsv',
+                'homology_mapping_flatfile' => '#homology_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.homology_id_map.tsv',
+                'output_file'               => '#wga_dumps_dir#/#hashed_mlss_id#/#orth_mlss_id#.#member_type#.wga_reuse.tsv',
+            },
+            -hive_capacity => 400,
+        },
+
+        {   -logic_name  => 'check_file_copy',
+            -module      => 'Bio::EnsEMBL::Hive::RunnableDB::Dummy',
+            -parameters  => { 'orth_wga_complete' => 1, },
+            -flow_into   => {
+                1 => WHEN( '#homology_dumps_shared_dir#' => 'copy_files_to_shared_loc' ),
+                2 => [ '?table_name=pipeline_wide_parameters' ],
```

Comment:
I don't think dummys flow anything on branch 2. You need to wire it to branch 1 instead.

Reply:
Ideally, you'd test that this works... 😉

Reply:
Yep, testing in new pipeline listed in the PR description stuff :+1: